spec/graphql/static_validation/rules/fields_will_merge_spec.rb

```diff
@@ -45,4 +45,27 @@
     ]
     assert_equal(expected_errors, error_messages)
   end
+
+  describe "when checking for directive conflicts" do
+    let(:query_string) {"
+      query getCheese {
+        cheese(id: 1) {
+          name,
+          ...F0,
+          ...F1
+        }
+      }
+      fragment F0 on Cheese {
+        name @include(if: true)
+      }
+      fragment F1 on Cheese {
+        name @skip(if: false)
```

Comment:
How would this behavior be different if this directive was instead: `@skip(if: true)`, would it get a validation error? (Am I overlooking an existing spec for this case?)